wezterm/install.ps1

```
@@ -0,0 +1,16 @@
+
+# wezterm config install — windows
+$scriptDir = Split-Path -Parent $MyInvocation.MyCommand.Path
+$configDir = Join-Path $env:USERPROFILE ".config\wezterm"
+
+New-Item -ItemType Directory -Path $configDir -Force | Out-Null
+Copy-Item (Join-Path $scriptDir "wezterm.lua") (Join-Path $configDir "wezterm.lua") -Force
+Write-Host "installed wezterm.lua to $configDir" -ForegroundColor Green
```

PSScriptAnalyzer (1.24.0)

[warning] File 'install.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)